antenna/throttler.py

```diff
@@ -223,29 +223,61 @@ def match_firefox_pre_57(data):
     )
 
 
-accept_all = [
+#: This accepts crash reports for all products
+ALL_PRODUCTS = []
+
+
+#: List of supported products; these have to match the ProductName of the
+#: incoming crash report
+MOZILLA_PRODUCTS = [
+    'Firefox',
+    'Fennec',
+    'Thunderbird',
+    'SeaMonkey'
```

Comment:
This is the list of products we currently support. "What about FennecAndroid?" you might say. To that, I reply, "Ugh. FennecAndroid is an abomination of MacGuyvering the system to bend it to the will of eldritch darkness." Then I might sigh, pause a bit, and recount my last expedition through the arctic waters of the processor rules which includes the run-in with the ProductRewrite rule which converts incoming crash reports with ProductName of Fennec to FennecAndroid. "It's an illusion! Clown shoes all the way down! Parrots! Too many parrots!" I scream! Roll 2d10 damage from psychic horror.